src/main.cpp

```diff
@@ -1200,7 +1200,7 @@ bool IsInitialBlockDownload()
     if (lockIBDState)
         return false;
     bool state = (chainActive.Height() < pindexBestHeader->nHeight - 24 * 6 ||
-            pindexBestHeader->GetBlockTime() < GetTime() - 24 * 60 * 60);
+            pindexBestHeader->GetBlockTime() < GetTime() - Params().MaxTipAge());
```

Comment:
would you mind to declare ```const CChainParams& chainparams = Params();``` at the beginning of the function and use chainparams.MaxTipAge() instead?
That will make passing CChainParams as an argument to IsInitialBlockDownload() simpler later.